cachedb-load-data/load_data.py

```diff
@@ -0,0 +1,111 @@
+import os
+
+import psycopg
+
+print("load_data.py")
+# get the POSTGRES_USR, POSTGRES_PW, POSTGRES_DBdata from environment variables or use default values
+POSTGRES_USR = os.getenv('POSTGRES_USR', 'cacheUser')
+POSTGRES_PW = os.getenv('POSTGRES_PW', 'cachePass')
+POSTGRES_DB = os.getenv('POSTGRES_DB', 'cacheDb')
+POSTGRES_HST = os.getenv('POSTGRES_HST', 'localhost')
+DATA_PATH = os.getenv('DATA_PATH', 'data')
+# print out the values
+print(f"POSTGRES_USR: {POSTGRES_USR}")
+print(f"POSTGRES_PW: {POSTGRES_PW}")
+print(f"POSTGRES_DB: {POSTGRES_DB}")
+print(f"POSTGRES_HST: {POSTGRES_HST}")
+
+TABLE_NAMES = [
+    'product_info',
+    'coupon_info',
+    'coupon_product',
+    'customer_info'
+]
+
+TABLE_COLUMNS_STRING = {
+    'product_info': 'product_id, name, category, sizes, vendor, description, buy_price, department',
+    'coupon_info': 'coupon_id, coupon_type, department, discount, how_many_products_required, start_date, end_date, product_mean_price, products_available',
+    'coupon_product': 'coupon_id, product_id',
+    'customer_info': 'customer_id, gender, age, mean_buy_price, total_coupons_used, mean_discount_received, unique_products_bought, unique_products_bought_with_coupons, total_items_bought'
+}
+
+
+def generate_insert_statement(table_name, table_columns_string):
+    sql = f"INSERT INTO {table_name} ({table_columns_string})"
+    columns = table_columns_string.split(',')
+    sql += " VALUES ("
+    for i in range(len(columns)):
+        sql += f"%({columns[i].strip()})s"
+        if i < len(columns) - 1:
+            sql += ", "
+    sql += ")"
+
+    return sql
+
+
+INSERT_STATEMENTS = {}
```

Comment:
not sure if `INSERT_STATEMENTS` should be treated as "constant". we can talk about it